lighthouse-labs/discord-shift-swap
# Add multi-server ability

## Change Summary

Added the ability to send messages in our mentor channels, rather than using a hard-coded channel ID for one server.

## Implementation Details


Changed the way the bot gets the channel name. Before, it was getting a channel ID from the env file, which would only allow for one channel ID to be used from one server. Now it uses the channel name to use for multiple servers. 

Changed from:
https://github.com/lighthouse-labs/discord-shift-swap/blob/8fd0781f0ca02e9724900fa32256248e680f3df5/slash/trade.js#L28

To:
https://github.com/lighthouse-labs/discord-shift-swap/blob/82f83edd838727481077f988730566892ef740af/slash/trade.js#L28

Also, changed the time limit on trades from 1m (for testing) to 15m (for production).

## Issue

Resolves #5 

## Estimation

Estimated: 1h
Actual:1h

## Manual Testing

Use the /trade command in a server that has the bot and see if it posts the message in the #shift-changes channel.
